modules/graceful-shutdown.adoc

```diff
@@ -77,3 +77,8 @@ Cluster administrators are responsible for ensuring a clean restart of their own
 ====
 
 . Shut off any cluster dependencies that are no longer needed, such as external storage or an LDAP server. Be sure to consult your vendor's documentation before doing so.
++
+[IMPORTANT]
+====
+If you deployed your cluster on a cloud-provider platform, do not shut down or suspend the associated cloud resources. If you delete the cloud resources of a suspended virtual machine, {product-title} might not restore successfully.
```

Comment:
@darshan-nagaraj one thing that stands out here is that there is a difference between shutting down/suspending the resources and deleting them. The reader might wonder how the two sentences relate.

To help with this, you could potentially update the first sentence to state "...do not shut down, suspend, or delete the associated cloud resources". Please ask for guidance from an SME on this point. Thanks!

Reply:
@pneedle-rh , @darshan-nagaraj Thank you for your effort!
It looks much better now. I think this covers the needs of the original customer's request, as this support case owner.

Reply:
@kazuhisya thank you for your feedback, this is very helpful!

Reply:
@kazuhisya Thank you. 
May I know are you okay with the changes I have made or should I consider @pneedle-rh suggestion ? Please clarify.

Reply:
@darshan-nagaraj Thanks.
Yes, e78c9ce is already good, but if possible, please incrude Paul's suggestions. It looks more elegant and friendly to understand.

Reply:
@kazuhisya Fixed.
@pneedle-rh Could you please merge the PR. Thanks